plugins/database/mssql/mssql_test.go

```diff
@@ -411,6 +458,30 @@ func assertCredsDoNotExist(t testing.TB, connURL, username, password string) {
 	}
 }
 
+func assertContainedDBCredsExist(t testing.TB, connURL, username string) {
+	t.Helper()
+	err := testContainedDBCredsExist(connURL, username)
+	if err == nil {
+		t.Fatalf("Able to drop user when it shouldn't")
+	}
+}
+
+func testContainedDBCredsExist(connURL, username string) error {
+	// Log in
+	db, err := sql.Open("mssql", connURL)
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	userQuery := `DECLARE @stmt nvarchar(max);
+			SET @stmt = 'DROP USER ' + QuoteName(@username);
+			EXEC(@stmt);`
+
+	_, err = db.Exec(userQuery, username)
```

Comment:
I think this style of sql statement requires a prepare or something, right? We should either do that, or use a non-prepared query like `DROP USER [%s]`.

Reply:
Added non prepared query in f782d30